sycl/source/detail/plugin.hpp

```diff
@@ -61,6 +61,8 @@ class plugin {
     if (MPiEnableTrace) {
       std::cout << ") ---> ";
       RT::printArgs(R);
+      RT::printOuts(Args...);
```

Comment:
We already print args on line 58, why do we need to print args again?

Reply:
this should only print output arguments (all pointer arguments) and the result

Reply:
On line 58 input arguments are printed: RT::printArgs(Args);

Here we print the **result**: RT::printArgs(R);
And all pointer arguments (because they are output arguments): RT::printOuts(Args...);

So there are no duplicate prints.